### DISCUSSION TITLE
Can't log into npmjs.com

### DISCUSSION THREAD

**Discussion by erispe** [community/community#137288] [npm]
### Select Topic Area

General

### Body

Hi, I can't log into the main website npmjs.com. I can't contact support as you have to be logged in.
I have reset my password but it still says my password is wrong after getting a confirmation email that it's been changed.
I tried emailing the press email address as it was the only way to contact the company I could find, but the email was bounced either by gmail or their webserver citing "Your message to press@npmjs.com has been blocked. See technical details below for more information. Response: Message bounced due to organizational settings."

What can I do? Do I have to create a second account just so I can talk to support?

**Comment by ebndev** [comment]
Hi @erispe, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `npm` category, as this topic may be more relevant there.

Have you tried filling out this [form](https://www.npmjs.com/support?inquire=account) on the Support Page? You don't have to be logged in to submit a request.

**Comment by dinosy817** [comment]
Here are some steps you can take to resolve the issue:

Clear Browser Cache: Sometimes, cached data can cause login issues. Clear your browser's cache and cookies, then try logging in again.
Try a Different Browser: If clearing the cache doesn't work, try using a different web browser or an incognito/private window.

Check for Typos: Make sure you're entering your email and password correctly, including any capitalization.
Use the 'Forgot Password' Feature Again: If you haven't already, try using the "Forgot Password?" link again to reset your password, ensuring you follow all steps correctly.

Check Spam/Junk Folder: Ensure that any emails from npm regarding your account (like password reset) haven't gone to your spam or junk folder.

Reach Out on Social Media: Sometimes, companies are responsive on platforms like Twitter. You could try reaching out to npm's official Twitter account for assistance.

Visit npm Community Forums: Check if there’s a community forum or user group for npm where you can ask for help.

Creating a Second Account: As a last resort, you could create a second account to contact npm support. However, be cautious about using multiple accounts, as it could lead to further complications.

**Comment by hiyeonKim** [comment]
I am having the same problem. Still cannot login after resetting the password for about 5 times now. Clearing cache didn't help. Can someone have a look at what's happening?

**Comment by Giachasidis** [comment]
Hey @erispe and others facing this login issue,

Here’s a quick checklist and some extra tips that might help:

1. **Password Reset Timing:** Sometimes password reset links or changes take a few minutes to propagate fully. Try waiting 10-15 minutes after resetting before logging in again.

2. **Use Incognito/Private Mode:** This avoids cached sessions or cookies messing with your login.

3. **Check Browser Extensions:** Some ad-blockers or privacy extensions can interfere with authentication flows. Try disabling them temporarily.

4. **Try Another Device or Network:** Sometimes corporate firewalls or VPNs cause issues.

5. **npm Support Contact Form:** You don’t need to be logged in to submit a support request here:
   [https://www.npmjs.com/support](https://www.npmjs.com/support)

6. **Social Media:** npm support is often responsive on Twitter at [[@npmjs](https://twitter.com/npmjs)](https://twitter.com/npmjs) — you can try reaching out there for help.

7. **Community Forums:** Check npm community forums or GitHub Discussions for any ongoing outage reports or similar issues.

8. **Avoid Creating Duplicate Accounts:** This can cause more problems down the line.

If none of these help, definitely submit a ticket via the support form above, including as much detail as possible (email, screenshots, timestamps).

Feel free to ping me if you want help drafting that message or troubleshooting further!

**Comment by homanp** [comment]
Password recovery emails are not working so impossible to reset anything at this point, been waiting for hours. None of the above measures work.

**Comment by Ben-Holme** [comment]
Same here. I even tried creating a new account. I got the one-time code but then something happened, and now I'm back to login page telling me username or password is invalid.

**Reply by homanp** [reply]
NPM is totally broken. Still haven't been able to access the account and support is not answering.

**Reply by homanp** [reply]
@Ben-Holme do you get a recovery email when trying to reset password?

**Comment by homanp** [comment]
@KaloudasDev recovery emails are not sent out after 2FA config. We are locked out of accounts. Any tips?

**Comment by cliangdev** [comment]
Same, keep getting "username or password was invalid". 
Reset using 2FA multiple times. npmjs is so buggy.

**Reply by IAM5K** [reply]
If anyone from `npm` is reading this please help us out because we are unable to publish our package due to login issues. 
We have tried all the above mentioned approaches. 
If there is an issue just  let us know with the expected time to fix it.

**Comment by afinzel** [comment]
Same

**Comment by Azure-Dotnet-Dev** [comment]
I had this issue but it was P.I.C.N.I.C (Problem In Chair, Not In Computer)
I use a password manager, which was entering my email into the "username" field. 
I knew my password was correct after numerous resets. 

Password manager complacency meant I wasn't paying attention to the specificity of the label "Username"

**Reply by RogierZeebregts** [reply]
> I had this issue but it was P.I.C.N.I.C (Problem In Chair, Not In Computer) I use a password manager, which was entering my email into the "username" field. I knew my password was correct after numerous resets.
> 
> Password manager complacency meant I wasn't paying attention to the specificity of the label "Username"

I can confirm this.... 🤦‍♂️

**Reply by kellend-git** [reply]
For anyone else hitting this the password reset email contains your username, after the "Hi " at the start

**Comment by laurefe-dev** [comment]
Same issue here. opened tickets through the form but it has been more than 36 hours and they haven't responded. This is crazy, I CANT WORK!

**Reply by RogierZeebregts** [reply]
Sure you're not also entering your email instead of your username?

**Comment by mshobeyri** [comment]
I do have a very same issue

**Comment by el-davo** [comment]
Still same issue, Reset password is never either  sent/received. what's going on?

**Comment by AllThingsSmitty** [comment]
I'm having this same problem.

I don't think it's a caching thing cuz it's been ages since I logged in. But I did clear cache and I've done 4-5 password resets at this point.

**Comment by SABBIR-H0SSAIN** [comment]
That definitely sounds frustrating, especially since you've already gone through the password reset process and received confirmation that it was successful.

Since you've confirmed that:
- the password reset email arrives,
- the password is successfully changed,
- but you still can't log in,

it doesn't sound like you're simply entering the wrong password. It could be an account-specific issue or something on npm's authentication side.

As for contacting support, you shouldn't need to create a second account just to get help. The `press@npmjs.com` address isn't intended for user support, so it's expected that messages to it may be rejected.

I'd recommend trying the official npm support channels that don't require you to be logged in, if available. If there isn't a public contact form, creating a temporary account solely to open a support request is probably your best option. When you contact them, include your username (not your password), explain that password resets complete successfully but authentication still fails, and mention that you can't access support because logging in is required.

Also, before contacting support, it may be worth trying:
- Logging in from a private/incognito browser window.
- Trying a different browser or device.
- Clearing your browser cookies for `npmjs.com`.
- Verifying that you're logging in with the correct username or email address.

Hopefully the support team can investigate whether there's an issue with your account specifically. Good luck—I hope it's something they can resolve quickly.
